data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	Once the audience is seated, an announcer again comes on the loudspeaker, reiterating the contestants that have qualified, though in an order that seems to be different than their ranking in the last event. Sensing your confusion, a Kimek sitting next to you with a translation device explains that the winning contestants from the previous event are resting. In the meantime they have been divided into six competing teams, each assigned a different color and with total scores as close to equal as possible. Apparently each participant will still be scored individually at the end of the event based on their specific contributions, in addition to their team's overall performance. As you sit waiting for the event to start, you are reminded of how out of place your are among the alien beings, and you realize you are quite likely the first human to ever witness these games.`
+			`	A slightly awkward half hour passes in which the uniqueness of your presence is more and more noted, before the thirty Saryds enter the arena and regain the crowd's attention. Holding what looks like fancy polo mallets, the members of each team are once more introduced to the crowd with a drone fly over, and you see that Ulmodi is on the blue team. After the introductions, several large spheres are placed at the center of the stadium in a circle and one member from each team stands around them.`
+			`	Once the game starts, you start to realize that it's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defences. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience. To your relief, especially when one of the large spheres heads your way, the nets are sturdy enough to prevent any accidents in the crowd.`
+			`	Eventually, a sphere finds its way into a hole, which lights up red for the color of the scoring team. Much of the crowd jumps up and screams, though you can't quite tell if everyone is a particular fan of anyone on the red team or if they're cheering simply because a point was finally scored. The sphere is shot back out automatically from a tube close to the hole and is immediately back in play.`
```

Comment:
```suggestion
			`	Eventually, a sphere finds its way into a hole, which lights up red for the color of the scoring team. Much of the crowd jumps up and screams, though you can't quite tell whether anyone particularly favors a team or if they're cheering simply because a point was finally scored. The sphere is shot back out automatically from a tube close to the hole and is immediately back in play.`
```